src/client.ts

```diff
@@ -122,10 +121,10 @@ export class SkillConfigurator implements Clova.SkillConfigurator {
    * Create firebase handler for dispatching request.
    * However, the contents are express.
    *
-   * @returns {Function}
+   * @returns
```

Comment:
As above, please add a return type to this function.
If there is a specific reason why no return type is included, let me know and include a reference as to why.

Reply:
I fixed it.